### ISSUE TITLE
422510 - CAD charged, wrong amt on showing on invoice in USD

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#867]
Amount that should show in USD is around $3060, not the $4,198 bc that was the $ amount in CAD.

Base price is not showing correctly when viewing invoice in CAD:

Should be $2,897:

http://www.206tours.com/tour206/index.htm#tours

**Comment by bradatkins** [comment]
This is correct in USD. The client made a 4198 payment, hence the refund when the discount is applied

![image](https://cloud.githubusercontent.com/assets/8440255/13589724/0d9c267a-e4a8-11e5-9023-6b9299ff8a6c.png)

2099 USD * 1.37 (Ex rate at the time) = 2875.63 CAD

I did a test order and everything appeared correctly (#422731)

I did notice that the balance due / refund total wasn't calculating correctly. I've fixed that.

![image](https://cloud.githubusercontent.com/assets/8440255/13589852/2beeea30-e4a9-11e5-9140-698f8888aca2.png)

**Comment by ali206** [comment]
The invoice is not correct in USD. The conversion on the CAD invoice is still going the wrong way and then on the USD invoice there is no conversion.

The charge was made in CAD as $4198 so on the USD invoice it should be less than $4198.00.

**Comment by ali206** [comment]
can you please urgently fix this conversion? Elavon's CAD conversion rate is .71196 and total charged was 3042.60

**Comment by bradatkins** [comment]
Fixed but I don't have that as the FX rate? I had it as .63? so it's gonna be messed up. Order total will be FX of .63 but payment as FX rate of .71196??

**Comment by ali206** [comment]
Yeah this one will always be messed up i guess. 

I just updated payment to be $3042.60 USD but it's not converting on CAD invoice now. 

Can you change xch rate in DB or is it set as is?

**Comment by bradatkins** [comment]
What do you want the FX rate to be?

**Comment by ali206** [comment]
.71196
